packages/office-ui-fabric-react/src/components/Separator/examples/Separator.Theming.Example.tsx

```diff
@@ -14,16 +14,11 @@ const theme: ITheme = createTheme({
 });
 
 const stackTokens: IStackTokens = { childrenGap: 12 };
+const content = 'Today';
 
-export class SeparatorThemingExample extends React.Component<{}, {}> {
-  public render(): JSX.Element {
-    const content = 'Today';
-
-    return (
-      <Stack tokens={stackTokens}>
-        <Text>Horizontal center aligned with custom theme</Text>
-        <Separator theme={theme}>{content}</Separator>
-      </Stack>
-    );
-  }
-}
+export const SeparatorThemingExample: React.FC = () => (
+  <Stack tokens={stackTokens}>
+    <Text>Horizontal center aligned with custom theme</Text>
+    <Separator theme={theme}>{content}</Separator>
```

Comment:
In this case you can just move the content inline since it's only used once.
```suggestion
    <Separator theme={theme}>Today</Separator>
```